### PR TITLE
bipblog.com

### LINKED ISSUE #119833: bipblog.com
### Issue URL (Ads)
 [http://bipblog.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fbipblog.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard DNS, <br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix https://github.com/AdguardTeam/AdguardFilters/issues/119833#issuecomment-1137254571

<details>
<summary>MEMO</summary>

!PC
!ad
`bipblog.com##.amachazl-box`
`bipblog.com###ent > iframe[src^="http://bipblog.com/"]`
!social
`bipblog.com##.ent_sub_block[style="margin-top: -65px;"]`
!
!mobile
`bipblog.com###article-contents > iframe[src^="http://bipblog.com/"]`

<details><summary>##.amachazl-box</summary>

</details><br/>

<details><summary>##.ent_sub_block[style="margin-top: -65px;"]x</summary>

</details><br/>

<details><summary>###ent > iframe[src^="http://bipblog.com/"]</summary>

</details><br/>

<details><summary>###article-contents > iframe[src^="http://bipblog.com/"]</summary>

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -1341,6 +1341,8 @@ bipblog.com###TopAmazon
 bipblog.com###BottomAmazon
 bipblog.com##.main_center_block[style^="margin"]
 bipblog.com##.main_left_block:nth-last-of-type(-n+4)
+bipblog.com##.amachazl-box
+bipblog.com###ent > iframe[src^="http://bipblog.com/"]
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:1345]: Same as below, but `||bipblog.com/bipblog.html` can be added to Annoyances > widget as this leads to third-party sites.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Alos change/add `||jin115.com^$removeparam=ref` in Trackparam filter to `$removeparam=ref,domain=jin115.com|bipblog.com`. This seems to be used on all these blogs.

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -46,6 +46,7 @@ dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn.com,slate.com,sankei.com,gam
 !
 careergarden.jp##.p-ad-wrapper
 sankeibiz.jp###textAd_wpics
+bipblog.com###article-contents > iframe[src^="http://bipblog.com/"]
```
[the remark this answers is not in the document]
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt:53]: i see.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:1356]: ok,done

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Please move `||img-c.net^$third-party` in JPF adserver to Annoyances > widget. This should have beend done already, our bad.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
I missed in the previous review but if Annoyance is enabled all those iframes are empty. You can change `||bipblog.com/bipblog.html` I requested previously to `||bipblog.com^$subdocument,~third-party` and also add `bipblog.com##iframe[src^="http://bipblog.com/"]` in Annoyances > widget. Sorry for extra work.

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**